central/policycategory/datastore/datastore_impl_test.go

```diff
@@ -38,82 +39,99 @@ func (s *PolicyCategoryDatastoreTestSuite) SetupTest() {
 
 	s.datastore = newWithoutDefaults(s.store, s.indexer, nil)
 
-	s.ctx = sac.WithGlobalAccessScopeChecker(context.Background(),
+	s.hasReadWritePolicyAccess = sac.WithGlobalAccessScopeChecker(context.Background(),
 		sac.AllowFixedScopes(
 			sac.AccessModeScopeKeys(storage.Access_READ_ACCESS, storage.Access_READ_WRITE_ACCESS),
 			sac.ResourceScopeKeys(resources.Policy)))
-}
-
-func (s *PolicyCategoryDatastoreTestSuite) TearDownTest() {
-	s.mockCtrl.Finish()
```

Comment:
Since go1.14 it's not required to call `mockCtrl.Finish()` separately if `*testing.T` is passed to the `NewController` function ([here's the reference](https://pkg.go.dev/github.com/golang/mock/gomock#Controller.Finish)). Hence, we can remove this not required code and simplify the test suites.